tools/autograd/gen_inplace_or_view_type.py

```diff
@@ -56,6 +59,7 @@
     "view_as_real",
     "_conj",
     "_neg_view",
+    "_nested_view_from_buffer",
```

Comment:
Just calling out my understanding for why we need this - nested tensors need to use autograd's view-replay logic, because they don't support arbitrary as_strided calls.

For normal view ops that gets called on nested tensors, that all Just Works because of this line:
```
std::function<at::Tensor(const at::Tensor&)> func=nullptr;
if (${is_view_with_metadata_change} || !self.unsafeGetTensorImpl()->support_as_strided()) {
  ${replay_view_func}
}
```

When the inputs to the view op are nested tensors, `support_as_strided()` is false, so we run the replay function. But `_nested_view_from_buffer()` takes in a normal tensor and returns a nested tensor, so we need to explicitly make sure the view replay stuff runs